src/Acme/BlogBundle/Controller/AdminPostController.php

```diff
@@ -69,8 +69,8 @@ public function manageAction(Request $request, $id)
                 $em->flush();
 
                 // New post : update post count
-                if ($post->getId() == null) {
-                    $this->updateCategoryPostCount($post->getCategory());
+                if ($post->getId() !== null) {
+                    $this->clearCache();
```

Comment:
That fix for update, but do nothing if it's a creation :/

So, the real fix is something like:

```php
if ($post->getId() == null) {
    $this->updateCategoryPostCount($post->getCategory());
} else {
    $this->clearCache();
}
```

What do you think?